managed-ledger/src/test/java/org/apache/bookkeeper/mledger/impl/ManagedCursorTest.java

```diff
@@ -974,6 +974,22 @@ void rewind() throws Exception {
         assertEquals(c1.getNumberOfEntries(), 2);
     }
 
+    @Test(timeOut = 20000)
+    void markDeleteThePositionTwice() throws Exception {
+        ManagedLedger ledger = factory.open("my_test_ledger", new ManagedLedgerConfig().setMaxEntriesPerLedger(10));
+        ManagedCursor cursor = ledger.openCursor("c1");
+        Position p1 = ledger.addEntry("dummy-entry-1".getBytes(Encoding));
+
+        assertEquals(cursor.getNumberOfEntries(), 1);
+
+        cursor.markDelete(p1);
+        assertFalse(cursor.hasMoreEntries());
+        assertEquals(cursor.getNumberOfEntries(), 0);
+
+        // markDelete p1 again
+        cursor.markDelete(p1);
```

Comment:
Do we need to verify the method `setAcknowledgedPosition` will not invoke?

Reply:
OK